File: programmers/javascript/level1/문자열다루기기본.js
const s = "1234";

function solution(s) {
  if (s.length === 4 || s.length === 6) {
    for (let i = 0; i < s.length; i++) {
      if (s[i] < "0" || s[i] > "9") {
        return false;
      }
    }
    return true;
  } else {
    return false;
  }
}

console.log(solution(s));

File: python/programmers/level1/문자열다루기기본.py
def isnumber(s):
    for i in range(len(s)):
        if s[i] < '0' or s[i] > '9':
            return False
    return True


def solution(s):
    if (len(s) == 4 or len(s) == 6) and isnumber(s):
        return True
    return False
